boards/adi/max32672evkit/doc/index.rst

```diff
@@ -0,0 +1,368 @@
+.. _max32672_evkit:
+
+MAX32672EVKIT
+#############
+
+Overview
+********
+The MAX32672 evaluation kit (EV kit) provides a platform
+for evaluating the capabilities of the MAX32672 microcontroller,
+which is a small, high-reliability, ultra-low power,
+32-bit microcontroller. The MAX32672 is a secure and
+cost-effective solution for motion/motor control, industrial
+sensors, and battery-powered medical devices and offers
+legacy designs an easy, cost-optimal upgrade path from
+8-bit or 16-bit microcontrollers.
+
+The Zephyr port is running on the MAX32672 MCU.
+
+.. image:: img/max32672evkit_img1.jpg
+   :align: center
+   :alt: MAX32672EVKIT Front
+
+.. image:: img/max32672evkit_img2.jpg
+   :align: center
+   :alt: MAX32672EVKIT Back
```

Comment:
drop the back picture (and associated file of course) as the added value is limited and this will save 10s of KBs from upstream git repo

Reply:
Correct, removed.
Thanks